src/utils.js

```diff
@@ -61,6 +61,18 @@ export default {
   makeElementDraggable(el) {
     const touchMode = 'ontouchstart' in document.documentElement;
     const elmnt = el;
+    let draggableEl;
+    if (document.getElementById(`${elmnt.id}-draggable`)) {
+      /* if present, the header is where you move the DIV from: */
+      draggableEl = document.getElementById(`${elmnt}-draggable`);
```

Comment:
Test is `elmnt.id`, assignment is `elmnt`. Is that correct

Reply:
By the way, I don't think that the `if (document.getElementById(`${elmnt.id}-draggable`))` clause is ever used. It possibly could be from infowindow and infowindow_list to remoge redundant code, but it is not.

Reply:
Yeah, that part is unnecessary and probably not functioning the way it is written.